ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -397,6 +397,42 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                 args[0]->get_data_ptr<const T>(), out[0]->get_data_ptr<T>(), element_count);
             break;
         }
+        case OP_TYPEID::CTCGreedyDecoder_v0:
+        {
+            const auto ctc_greedy_dec = static_cast<const op::v0::CTCGreedyDecoder*>(&node);
+            auto type = node.get_input_element_type(1);
+            if (type == element::bf16)
+            {
+                reference::ctc_greedy_decoder<bfloat16>(args[0]->get_data_ptr<bfloat16>(),
+                                                        args[1]->get_data_ptr<bfloat16>(),
+                                                        out[0]->get_data_ptr<bfloat16>(),
+                                                        args[0]->get_shape(),
+                                                        args[1]->get_shape(),
+                                                        out[0]->get_shape(),
+                                                        ctc_greedy_dec->get_ctc_merge_repeated());
```

Comment:
But do we really need to specify type here? What about:
```suggestion
                reference::ctc_greedy_decoder<T>(args[0]->get_data_ptr<const T>(),
                                                        args[1]->get_data_ptr<const T>(),
                                                        out[0]->get_data_ptr<T>(),
                                                        args[0]->get_shape(),
                                                        args[1]->get_shape(),
                                                        out[0]->get_shape(),
                                                        ctc_greedy_dec->get_ctc_merge_repeated());
```

Reply:
You're totally right, my bad.